src/services/requetsToEndpoint.ts

```diff
@@ -0,0 +1,38 @@
+import { getToken } from "./generateToken";
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **nitpick (typo):** The file name `requetsToEndpoint.ts` seems to be a typo. Consider renaming it to `requestsToEndpoint.ts` for clarity and consistency.